validate_rpm_specs.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import argparse
+import glob
+import os
+import sys
+
+from lib import exception
+from lib.utils import is_package_installed
+from lib.utils import run_command
+
+def validate_rpm_spec(spec_file_path):
+
+    try:
+        run_command("rpmlint -v %s" % spec_file_path)
+    except exception.SubprocessError as e:
+        #pylint: disable=no-member
+        print("validation of rpm spec file %s failed, output: %s" % (spec_file_path, e.stdout))
+        return
+
+def validate_rpm_specs(base_dir):
+    """
+    Validate specification files of rpm packages built by Open Power Host OS
+    """
+
+    packages = os.listdir(base_dir)
+
+    for package in packages:
+        files = glob.glob("{base_dir}/{package_name}/*/*/{package_name}.spec".format(
+            base_dir=base_dir,package_name=package))
+        for _file in files:
+            validate_rpm_spec(_file)
+
+def parse_cli_options():
+    """
+    Parse CLI options
+
+    Returns:
+        Namespace: CLI options. Valid attributes: packages_metadata_base_dir
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-p', '--packages-metadata-base-dir', dest='packages_metadata_base_dir',
+                        required=True, help='packages metadata base directory path')
```

Comment:
I think `--specs-dir` would reflect more the meaning of the parameter since it's the base directory of where spec files reside.  That would also make `validate_rpm_specs.py` a more generic tool, not tailored only for Host OS packages.